2014/04/28 07:17:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "hola/index".' in M:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#1 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:17:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "hola/index".' in M:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#1 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:17:53 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "hola/index".' in M:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#1 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:17:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "hola/index".' in M:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#1 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:17:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "hola/index".' in M:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#1 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:30:07 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:30:09 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:31:49 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:31:50 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:31:51 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:32:04 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:42:47 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:42:59 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:42:59 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:43:00 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:43:00 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:43:00 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:43:00 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:43:01 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:43:01 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:43:01 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:43:02 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:43:02 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:43:02 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:43:05 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:43:05 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:43:06 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:43:06 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:43:06 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:43:08 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:43:08 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:43:08 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:43:08 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:43:09 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:43:09 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:43:13 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 07:43:14 [error] [exception.CException] exception 'CException' with message 'Users and its behaviors do not have a method or closure named "findAll".' in M:\xampp\htdocs\yii\framework\base\CComponent.php:266
Stack trace:
#0 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): CComponent->__call('findAll', Array)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(14): Users->findAll()
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('hola/index')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/hola/index
---
2014/04/28 08:05:47 [error] [php] Undefined variable: twitter (M:\xampp\htdocs\yii\facilito\protected\views\site\index.php:4)
Stack trace:
#0 M:\xampp\htdocs\yii\framework\web\CController.php(869): SiteController->renderFile()
#1 M:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 M:\xampp\htdocs\yii\facilito\protected\controllers\SiteController.php(32): SiteController->render()
#3 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#4 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 M:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 M:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 M:\xampp\htdocs\yii\facilito\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/facilito/site/index.html
in M:\xampp\htdocs\yii\facilito\protected\views\site\index.php (4)
in M:\xampp\htdocs\yii\facilito\protected\controllers\SiteController.php (32)
in M:\xampp\htdocs\yii\facilito\index.php (13)
2014/04/28 08:06:03 [error] [php] Undefined variable: twitter (M:\xampp\htdocs\yii\facilito\protected\views\site\index.php:4)
Stack trace:
#0 M:\xampp\htdocs\yii\framework\web\CController.php(869): SiteController->renderFile()
#1 M:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 M:\xampp\htdocs\yii\facilito\protected\controllers\SiteController.php(32): SiteController->render()
#3 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#4 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 M:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 M:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 M:\xampp\htdocs\yii\facilito\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/facilito/site/index.html
in M:\xampp\htdocs\yii\facilito\protected\views\site\index.php (4)
in M:\xampp\htdocs\yii\facilito\protected\controllers\SiteController.php (32)
in M:\xampp\htdocs\yii\facilito\index.php (13)
2014/04/28 08:06:04 [error] [php] Undefined variable: twitter (M:\xampp\htdocs\yii\facilito\protected\views\site\index.php:4)
Stack trace:
#0 M:\xampp\htdocs\yii\framework\web\CController.php(869): SiteController->renderFile()
#1 M:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 M:\xampp\htdocs\yii\facilito\protected\controllers\SiteController.php(32): SiteController->render()
#3 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#4 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 M:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 M:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 M:\xampp\htdocs\yii\facilito\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/facilito/site/index.html
in M:\xampp\htdocs\yii\facilito\protected\views\site\index.php (4)
in M:\xampp\htdocs\yii\facilito\protected\controllers\SiteController.php (32)
in M:\xampp\htdocs\yii\facilito\index.php (13)
2014/04/28 08:06:06 [error] [php] Undefined variable: twitter (M:\xampp\htdocs\yii\facilito\protected\views\site\index.php:4)
Stack trace:
#0 M:\xampp\htdocs\yii\framework\web\CController.php(869): SiteController->renderFile()
#1 M:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 M:\xampp\htdocs\yii\facilito\protected\controllers\SiteController.php(32): SiteController->render()
#3 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#4 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 M:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 M:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 M:\xampp\htdocs\yii\facilito\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/facilito/
in M:\xampp\htdocs\yii\facilito\protected\views\site\index.php (4)
in M:\xampp\htdocs\yii\facilito\protected\controllers\SiteController.php (32)
in M:\xampp\htdocs\yii\facilito\index.php (13)
2014/04/28 08:06:10 [error] [php] Undefined variable: twitter (M:\xampp\htdocs\yii\facilito\protected\views\site\index.php:4)
Stack trace:
#0 M:\xampp\htdocs\yii\framework\web\CController.php(869): SiteController->renderFile()
#1 M:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 M:\xampp\htdocs\yii\facilito\protected\controllers\SiteController.php(32): SiteController->render()
#3 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#4 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 M:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 M:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 M:\xampp\htdocs\yii\facilito\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/facilito/site/index.html
in M:\xampp\htdocs\yii\facilito\protected\views\site\index.php (4)
in M:\xampp\htdocs\yii\facilito\protected\controllers\SiteController.php (32)
in M:\xampp\htdocs\yii\facilito\index.php (13)
2014/04/28 08:06:12 [error] [php] Undefined variable: twitter (M:\xampp\htdocs\yii\facilito\protected\views\site\index.php:4)
Stack trace:
#0 M:\xampp\htdocs\yii\framework\web\CController.php(869): SiteController->renderFile()
#1 M:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 M:\xampp\htdocs\yii\facilito\protected\controllers\SiteController.php(32): SiteController->render()
#3 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#4 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 M:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 M:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 M:\xampp\htdocs\yii\facilito\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/facilito/site/index.html
in M:\xampp\htdocs\yii\facilito\protected\views\site\index.php (4)
in M:\xampp\htdocs\yii\facilito\protected\controllers\SiteController.php (32)
in M:\xampp\htdocs\yii\facilito\index.php (13)
2014/04/28 08:06:13 [error] [php] Undefined variable: twitter (M:\xampp\htdocs\yii\facilito\protected\views\site\index.php:4)
Stack trace:
#0 M:\xampp\htdocs\yii\framework\web\CController.php(869): SiteController->renderFile()
#1 M:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 M:\xampp\htdocs\yii\facilito\protected\controllers\SiteController.php(32): SiteController->render()
#3 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#4 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 M:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 M:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 M:\xampp\htdocs\yii\facilito\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/facilito/site/index.html
in M:\xampp\htdocs\yii\facilito\protected\views\site\index.php (4)
in M:\xampp\htdocs\yii\facilito\protected\controllers\SiteController.php (32)
in M:\xampp\htdocs\yii\facilito\index.php (13)
2014/04/28 08:13:58 [error] [php] Undefined variable: twitterx (M:\xampp\htdocs\yii\facilito\protected\views\hola\index.php:4)
Stack trace:
#0 M:\xampp\htdocs\yii\framework\web\CController.php(869): HolaController->renderFile()
#1 M:\xampp\htdocs\yii\framework\web\CController.php(782): HolaController->renderPartial()
#2 M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php(19): HolaController->render()
#3 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): HolaController->actionIndex()
#4 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 M:\xampp\htdocs\yii\framework\web\CController.php(286): HolaController->runAction()
#6 M:\xampp\htdocs\yii\framework\web\CController.php(265): HolaController->runActionWithFilters()
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): HolaController->run()
#8 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 M:\xampp\htdocs\yii\facilito\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/facilito/hola/index
in M:\xampp\htdocs\yii\facilito\protected\views\hola\index.php (4)
in M:\xampp\htdocs\yii\facilito\protected\controllers\HolaController.php (19)
in M:\xampp\htdocs\yii\facilito\index.php (13)
2014/04/28 19:37:47 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('nombre')
#1 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2529): CActiveRecord->__get('nombre')
#2 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2252): CHtml::resolveValue(Object(Diente), 'nombre')
#3 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1434): CHtml::activeInputField('text', Object(Diente), 'nombre', Array)
#4 M:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(725): CHtml::activeTextField(Object(Diente), 'nombre', Array)
#5 M:\xampp\htdocs\yii\facilito\protected\views\diente\create.php(14): CActiveForm->textField(Object(Diente), 'nombre')
#6 M:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('M:\xampp\htdocs...')
#7 M:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('M:\xampp\htdocs...', Array, true)
#8 M:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('M:\xampp\htdocs...', Array, true)
#9 M:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#10 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(17): CController->render('create', Array)
#11 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionCreate()
#12 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#16 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/create')
#17 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/facilito/diente/create
---
2014/04/28 19:39:28 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('nombre')
#1 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2529): CActiveRecord->__get('nombre')
#2 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2252): CHtml::resolveValue(Object(Diente), 'nombre')
#3 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1434): CHtml::activeInputField('text', Object(Diente), 'nombre', Array)
#4 M:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(725): CHtml::activeTextField(Object(Diente), 'nombre', Array)
#5 M:\xampp\htdocs\yii\facilito\protected\views\diente\create.php(13): CActiveForm->textField(Object(Diente), 'nombre')
#6 M:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('M:\xampp\htdocs...')
#7 M:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('M:\xampp\htdocs...', Array, true)
#8 M:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('M:\xampp\htdocs...', Array, true)
#9 M:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#10 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(17): CController->render('create', Array)
#11 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionCreate()
#12 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#16 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/create')
#17 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/facilito/diente/create
---
2014/04/28 19:41:15 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('nombre')
#1 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2529): CActiveRecord->__get('nombre')
#2 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2252): CHtml::resolveValue(Object(Diente), 'nombre')
#3 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1434): CHtml::activeInputField('text', Object(Diente), 'nombre', Array)
#4 M:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(725): CHtml::activeTextField(Object(Diente), 'nombre', Array)
#5 M:\xampp\htdocs\yii\facilito\protected\views\diente\create.php(13): CActiveForm->textField(Object(Diente), 'nombre')
#6 M:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('M:\xampp\htdocs...')
#7 M:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('M:\xampp\htdocs...', Array, true)
#8 M:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('M:\xampp\htdocs...', Array, true)
#9 M:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#10 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(17): CController->render('create', Array)
#11 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionCreate()
#12 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#16 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/create')
#17 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/facilito/diente/create
---
2014/04/28 19:44:46 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('nombre')
#1 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2529): CActiveRecord->__get('nombre')
#2 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2252): CHtml::resolveValue(Object(Diente), 'nombre')
#3 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1434): CHtml::activeInputField('text', Object(Diente), 'nombre', Array)
#4 M:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(725): CHtml::activeTextField(Object(Diente), 'nombre', Array)
#5 M:\xampp\htdocs\yii\facilito\protected\views\diente\create.php(13): CActiveForm->textField(Object(Diente), 'nombre')
#6 M:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('M:\xampp\htdocs...')
#7 M:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('M:\xampp\htdocs...', Array, true)
#8 M:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('M:\xampp\htdocs...', Array, true)
#9 M:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#10 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(17): CController->render('create', Array)
#11 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionCreate()
#12 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#16 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/create')
#17 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/facilito/diente/create
---
2014/04/28 19:44:48 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('nombre')
#1 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2529): CActiveRecord->__get('nombre')
#2 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2252): CHtml::resolveValue(Object(Diente), 'nombre')
#3 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1434): CHtml::activeInputField('text', Object(Diente), 'nombre', Array)
#4 M:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(725): CHtml::activeTextField(Object(Diente), 'nombre', Array)
#5 M:\xampp\htdocs\yii\facilito\protected\views\diente\create.php(13): CActiveForm->textField(Object(Diente), 'nombre')
#6 M:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('M:\xampp\htdocs...')
#7 M:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('M:\xampp\htdocs...', Array, true)
#8 M:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('M:\xampp\htdocs...', Array, true)
#9 M:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#10 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(17): CController->render('create', Array)
#11 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionCreate()
#12 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#16 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/create')
#17 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/facilito/diente/create
---
2014/04/28 19:44:49 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('nombre')
#1 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2529): CActiveRecord->__get('nombre')
#2 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2252): CHtml::resolveValue(Object(Diente), 'nombre')
#3 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1434): CHtml::activeInputField('text', Object(Diente), 'nombre', Array)
#4 M:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(725): CHtml::activeTextField(Object(Diente), 'nombre', Array)
#5 M:\xampp\htdocs\yii\facilito\protected\views\diente\create.php(13): CActiveForm->textField(Object(Diente), 'nombre')
#6 M:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('M:\xampp\htdocs...')
#7 M:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('M:\xampp\htdocs...', Array, true)
#8 M:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('M:\xampp\htdocs...', Array, true)
#9 M:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#10 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(17): CController->render('create', Array)
#11 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionCreate()
#12 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#16 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/create')
#17 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/facilito/diente/create
---
2014/04/28 19:44:49 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('nombre')
#1 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2529): CActiveRecord->__get('nombre')
#2 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2252): CHtml::resolveValue(Object(Diente), 'nombre')
#3 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1434): CHtml::activeInputField('text', Object(Diente), 'nombre', Array)
#4 M:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(725): CHtml::activeTextField(Object(Diente), 'nombre', Array)
#5 M:\xampp\htdocs\yii\facilito\protected\views\diente\create.php(13): CActiveForm->textField(Object(Diente), 'nombre')
#6 M:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('M:\xampp\htdocs...')
#7 M:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('M:\xampp\htdocs...', Array, true)
#8 M:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('M:\xampp\htdocs...', Array, true)
#9 M:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#10 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(17): CController->render('create', Array)
#11 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionCreate()
#12 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#16 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/create')
#17 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/facilito/diente/create
---
2014/04/28 19:44:49 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('nombre')
#1 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2529): CActiveRecord->__get('nombre')
#2 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2252): CHtml::resolveValue(Object(Diente), 'nombre')
#3 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1434): CHtml::activeInputField('text', Object(Diente), 'nombre', Array)
#4 M:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(725): CHtml::activeTextField(Object(Diente), 'nombre', Array)
#5 M:\xampp\htdocs\yii\facilito\protected\views\diente\create.php(13): CActiveForm->textField(Object(Diente), 'nombre')
#6 M:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('M:\xampp\htdocs...')
#7 M:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('M:\xampp\htdocs...', Array, true)
#8 M:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('M:\xampp\htdocs...', Array, true)
#9 M:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#10 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(17): CController->render('create', Array)
#11 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionCreate()
#12 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#16 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/create')
#17 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/facilito/diente/create
---
2014/04/28 19:45:35 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('nombre')
#1 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2529): CActiveRecord->__get('nombre')
#2 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2252): CHtml::resolveValue(Object(Diente), 'nombre')
#3 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1434): CHtml::activeInputField('text', Object(Diente), 'nombre', Array)
#4 M:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(725): CHtml::activeTextField(Object(Diente), 'nombre', Array)
#5 M:\xampp\htdocs\yii\facilito\protected\views\diente\create.php(13): CActiveForm->textField(Object(Diente), 'nombre')
#6 M:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('M:\xampp\htdocs...')
#7 M:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('M:\xampp\htdocs...', Array, true)
#8 M:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('M:\xampp\htdocs...', Array, true)
#9 M:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#10 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(17): CController->render('create', Array)
#11 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionCreate()
#12 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#16 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/create')
#17 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/facilito/diente/create
---
2014/04/28 19:45:42 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('nombre')
#1 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2529): CActiveRecord->__get('nombre')
#2 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2252): CHtml::resolveValue(Object(Diente), 'nombre')
#3 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1434): CHtml::activeInputField('text', Object(Diente), 'nombre', Array)
#4 M:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(725): CHtml::activeTextField(Object(Diente), 'nombre', Array)
#5 M:\xampp\htdocs\yii\facilito\protected\views\diente\create.php(13): CActiveForm->textField(Object(Diente), 'nombre')
#6 M:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('M:\xampp\htdocs...')
#7 M:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('M:\xampp\htdocs...', Array, true)
#8 M:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('M:\xampp\htdocs...', Array, true)
#9 M:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#10 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(17): CController->render('create', Array)
#11 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionCreate()
#12 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#16 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/create')
#17 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/facilito/diente/create
---
2014/04/28 19:48:08 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(145): CComponent->__get('nombre')
#1 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2529): CActiveRecord->__get('nombre')
#2 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2252): CHtml::resolveValue(Object(Diente), 'nombre')
#3 M:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1434): CHtml::activeInputField('text', Object(Diente), 'nombre', Array)
#4 M:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(725): CHtml::activeTextField(Object(Diente), 'nombre', Array)
#5 M:\xampp\htdocs\yii\facilito\protected\views\diente\create.php(13): CActiveForm->textField(Object(Diente), 'nombre')
#6 M:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('M:\xampp\htdocs...')
#7 M:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('M:\xampp\htdocs...', Array, true)
#8 M:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('M:\xampp\htdocs...', Array, true)
#9 M:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#10 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(17): CController->render('create', Array)
#11 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionCreate()
#12 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#14 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#15 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#16 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/create')
#17 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#18 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#19 {main}
REQUEST_URI=/yii/facilito/diente/create
---
2014/04/28 19:48:18 [error] [php] Undefined variable: dientes (M:\xampp\htdocs\yii\facilito\protected\views\diente\index.php:19)
Stack trace:
#0 M:\xampp\htdocs\yii\framework\web\CController.php(869): DienteController->renderFile()
#1 M:\xampp\htdocs\yii\framework\web\CController.php(782): DienteController->renderPartial()
#2 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(10): DienteController->render()
#3 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#4 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 M:\xampp\htdocs\yii\framework\web\CController.php(286): DienteController->runAction()
#6 M:\xampp\htdocs\yii\framework\web\CController.php(265): DienteController->runActionWithFilters()
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DienteController->run()
#8 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 M:\xampp\htdocs\yii\facilito\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/facilito/diente/
in M:\xampp\htdocs\yii\facilito\protected\views\diente\index.php (19)
in M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php (10)
in M:\xampp\htdocs\yii\facilito\index.php (13)
2014/04/28 19:49:54 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('nombre', 'molar1')
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('nombre', 'molar1')
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:49:56 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('nombre', 'molar1')
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('nombre', 'molar1')
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:53:12 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('nombre', 'molar1')
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('nombre', 'molar1')
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:53:14 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('nombre', 'molar1')
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('nombre', 'molar1')
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:53:14 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('nombre', 'molar1')
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('nombre', 'molar1')
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:53:15 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('nombre', 'molar1')
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('nombre', 'molar1')
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:53:27 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('nombre', 'molar1')
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('nombre', 'molar1')
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:53:50 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('nombre', 'molar1')
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('nombre', 'molar1')
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:53:51 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('nombre', 'molar1')
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('nombre', 'molar1')
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:54:29 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('nombre', 'molar2')
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('nombre', 'molar2')
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:54:30 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('nombre', 'molar2')
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('nombre', 'molar2')
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:54:31 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('nombre', 'molar2')
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('nombre', 'molar2')
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:54:31 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('nombre', 'molar2')
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('nombre', 'molar2')
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:54:32 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('nombre', 'molar2')
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('nombre', 'molar2')
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:54:32 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('nombre', 'molar2')
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('nombre', 'molar2')
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:54:32 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('nombre', 'molar2')
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('nombre', 'molar2')
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:57:04 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('nombre', 'molar2')
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('nombre', 'molar2')
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:57:05 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('nombre', 'molar2')
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('nombre', 'molar2')
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:57:06 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('nombre', 'molar2')
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('nombre', 'molar2')
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:57:06 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('nombre', 'molar2')
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('nombre', 'molar2')
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:57:06 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('nombre', 'molar2')
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('nombre', 'molar2')
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:57:06 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('nombre', 'molar2')
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('nombre', 'molar2')
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:57:53 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('nombre', 'molar2')
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('nombre', 'molar2')
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:57:55 [error] [exception.CException] exception 'CException' with message 'Property "Diente.nombre" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('nombre', 'molar2')
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('nombre', 'molar2')
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:58:25 [error] [exception.CException] exception 'CException' with message 'Property "Diente.estado" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('estado', 2)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('estado', 2)
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:58:26 [error] [exception.CException] exception 'CException' with message 'Property "Diente.estado" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('estado', 2)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('estado', 2)
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:58:26 [error] [exception.CException] exception 'CException' with message 'Property "Diente.estado" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('estado', 2)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('estado', 2)
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:58:27 [error] [exception.CException] exception 'CException' with message 'Property "Diente.estado" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('estado', 2)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('estado', 2)
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:58:27 [error] [exception.CException] exception 'CException' with message 'Property "Diente.estado" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('estado', 2)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('estado', 2)
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:58:27 [error] [exception.CException] exception 'CException' with message 'Property "Diente.estado" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('estado', 2)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('estado', 2)
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:58:27 [error] [exception.CException] exception 'CException' with message 'Property "Diente.estado" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('estado', 2)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('estado', 2)
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 19:59:25 [error] [exception.CException] exception 'CException' with message 'Property "Diente.estado" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('estado', 2)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('estado', 2)
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 20:00:09 [error] [exception.CException] exception 'CException' with message 'Property "Diente.estado" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('estado', 2)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('estado', 2)
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 20:00:10 [error] [exception.CException] exception 'CException' with message 'Property "Diente.estado" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('estado', 2)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('estado', 2)
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 20:00:10 [error] [exception.CException] exception 'CException' with message 'Property "Diente.estado" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('estado', 2)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('estado', 2)
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 20:00:14 [error] [exception.CException] exception 'CException' with message 'Property "Diente.estado" is not defined.' in M:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(161): CComponent->__set('estado', 2)
#1 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(9): CActiveRecord->__set('estado', 2)
#2 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#3 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('')
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente')
#8 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#9 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/yii/facilito/diente/
---
2014/04/28 20:00:36 [error] [php] Undefined variable: dientes (M:\xampp\htdocs\yii\facilito\protected\views\diente\index.php:19)
Stack trace:
#0 M:\xampp\htdocs\yii\framework\web\CController.php(869): DienteController->renderFile()
#1 M:\xampp\htdocs\yii\framework\web\CController.php(782): DienteController->renderPartial()
#2 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(16): DienteController->render()
#3 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionIndex()
#4 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 M:\xampp\htdocs\yii\framework\web\CController.php(286): DienteController->runAction()
#6 M:\xampp\htdocs\yii\framework\web\CController.php(265): DienteController->runActionWithFilters()
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DienteController->run()
#8 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 M:\xampp\htdocs\yii\facilito\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/facilito/diente/
in M:\xampp\htdocs\yii\facilito\protected\views\diente\index.php (19)
in M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php (16)
in M:\xampp\htdocs\yii\facilito\index.php (13)
2014/04/28 20:04:58 [error] [php] Undefined variable: POST (M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php:24)
Stack trace:
#0 M:\xampp\htdocs\yii\framework\web\CController.php(286): DienteController->runAction()
#1 M:\xampp\htdocs\yii\framework\web\CController.php(265): DienteController->runActionWithFilters()
#2 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DienteController->run()
#3 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#5 M:\xampp\htdocs\yii\facilito\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/facilito/diente/create
in M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php (24)
in M:\xampp\htdocs\yii\facilito\index.php (13)
2014/04/28 20:05:13 [error] [php] Undefined variable: POST (M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php:24)
Stack trace:
#0 M:\xampp\htdocs\yii\framework\web\CController.php(286): DienteController->runAction()
#1 M:\xampp\htdocs\yii\framework\web\CController.php(265): DienteController->runActionWithFilters()
#2 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DienteController->run()
#3 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#5 M:\xampp\htdocs\yii\facilito\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/facilito/diente/create
in M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php (24)
in M:\xampp\htdocs\yii\facilito\index.php (13)
2014/04/28 20:05:14 [error] [php] Undefined variable: POST (M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php:24)
Stack trace:
#0 M:\xampp\htdocs\yii\framework\web\CController.php(286): DienteController->runAction()
#1 M:\xampp\htdocs\yii\framework\web\CController.php(265): DienteController->runActionWithFilters()
#2 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DienteController->run()
#3 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#5 M:\xampp\htdocs\yii\facilito\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/facilito/diente/create
in M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php (24)
in M:\xampp\htdocs\yii\facilito\index.php (13)
2014/04/28 20:05:14 [error] [php] Undefined variable: POST (M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php:24)
Stack trace:
#0 M:\xampp\htdocs\yii\framework\web\CController.php(286): DienteController->runAction()
#1 M:\xampp\htdocs\yii\framework\web\CController.php(265): DienteController->runActionWithFilters()
#2 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DienteController->run()
#3 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#5 M:\xampp\htdocs\yii\facilito\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/facilito/diente/create
in M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php (24)
in M:\xampp\htdocs\yii\facilito\index.php (13)
2014/04/28 20:05:14 [error] [php] Undefined variable: POST (M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php:24)
Stack trace:
#0 M:\xampp\htdocs\yii\framework\web\CController.php(286): DienteController->runAction()
#1 M:\xampp\htdocs\yii\framework\web\CController.php(265): DienteController->runActionWithFilters()
#2 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DienteController->run()
#3 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#5 M:\xampp\htdocs\yii\facilito\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/facilito/diente/create
in M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php (24)
in M:\xampp\htdocs\yii\facilito\index.php (13)
2014/04/28 20:05:37 [error] [php] Undefined variable: POST (M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php:24)
Stack trace:
#0 M:\xampp\htdocs\yii\framework\web\CController.php(286): DienteController->runAction()
#1 M:\xampp\htdocs\yii\framework\web\CController.php(265): DienteController->runActionWithFilters()
#2 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DienteController->run()
#3 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#5 M:\xampp\htdocs\yii\facilito\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/facilito/diente/create
in M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php (24)
in M:\xampp\htdocs\yii\facilito\index.php (13)
2014/04/28 20:05:41 [error] [php] Undefined variable: POST (M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php:24)
Stack trace:
#0 M:\xampp\htdocs\yii\framework\web\CController.php(286): DienteController->runAction()
#1 M:\xampp\htdocs\yii\framework\web\CController.php(265): DienteController->runActionWithFilters()
#2 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): DienteController->run()
#3 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#5 M:\xampp\htdocs\yii\facilito\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/facilito/diente/create
in M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php (24)
in M:\xampp\htdocs\yii\facilito\index.php (13)
2014/04/28 21:33:45 [error] [exception.CDbException] exception 'CDbException' with message 'Column name must be either a string or an array.' in M:\xampp\htdocs\yii\framework\db\schema\CDbCommandBuilder.php:828
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\schema\CDbCommandBuilder.php(621): CDbCommandBuilder->createInCondition(Object(CMysqlTableSchema), NULL, Array, '`t`.')
#1 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1485): CDbCommandBuilder->createPkCriteria(Object(CMysqlTableSchema), '2', '', Array, '`t`.')
#2 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(48): CActiveRecord->findByPk('2')
#3 [internal function]: DienteController->actionUpdate('2')
#4 M:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(DienteController), Array)
#5 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(DienteController), Object(ReflectionMethod), Array)
#6 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#10 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/update')
#11 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/yii/facilito/diente/update/2.html
HTTP_REFERER=http://localhost/yii/facilito/diente/
---
2014/04/28 21:33:55 [error] [exception.CDbException] exception 'CDbException' with message 'Column name must be either a string or an array.' in M:\xampp\htdocs\yii\framework\db\schema\CDbCommandBuilder.php:828
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\schema\CDbCommandBuilder.php(621): CDbCommandBuilder->createInCondition(Object(CMysqlTableSchema), NULL, Array, '`t`.')
#1 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1485): CDbCommandBuilder->createPkCriteria(Object(CMysqlTableSchema), '1', '', Array, '`t`.')
#2 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(48): CActiveRecord->findByPk('1')
#3 [internal function]: DienteController->actionUpdate('1')
#4 M:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(DienteController), Array)
#5 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(DienteController), Object(ReflectionMethod), Array)
#6 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#10 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/update')
#11 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/yii/facilito/diente/update/1.html
HTTP_REFERER=http://localhost/yii/facilito/diente/
---
2014/04/28 21:34:51 [error] [exception.CDbException] exception 'CDbException' with message 'Column name must be either a string or an array.' in M:\xampp\htdocs\yii\framework\db\schema\CDbCommandBuilder.php:828
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\schema\CDbCommandBuilder.php(621): CDbCommandBuilder->createInCondition(Object(CMysqlTableSchema), NULL, Array, '`t`.')
#1 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1485): CDbCommandBuilder->createPkCriteria(Object(CMysqlTableSchema), '1', '', Array, '`t`.')
#2 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(48): CActiveRecord->findByPk('1')
#3 [internal function]: DienteController->actionUpdate('1')
#4 M:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(DienteController), Array)
#5 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(DienteController), Object(ReflectionMethod), Array)
#6 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#10 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/update')
#11 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/yii/facilito/diente/update/1.html
HTTP_REFERER=http://localhost/yii/facilito/diente/
---
2014/04/28 21:34:51 [error] [exception.CDbException] exception 'CDbException' with message 'Column name must be either a string or an array.' in M:\xampp\htdocs\yii\framework\db\schema\CDbCommandBuilder.php:828
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\schema\CDbCommandBuilder.php(621): CDbCommandBuilder->createInCondition(Object(CMysqlTableSchema), NULL, Array, '`t`.')
#1 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1485): CDbCommandBuilder->createPkCriteria(Object(CMysqlTableSchema), '1', '', Array, '`t`.')
#2 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(48): CActiveRecord->findByPk('1')
#3 [internal function]: DienteController->actionUpdate('1')
#4 M:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(DienteController), Array)
#5 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(DienteController), Object(ReflectionMethod), Array)
#6 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#10 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/update')
#11 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/yii/facilito/diente/update/1.html
HTTP_REFERER=http://localhost/yii/facilito/diente/
---
2014/04/28 21:34:53 [error] [exception.CDbException] exception 'CDbException' with message 'Column name must be either a string or an array.' in M:\xampp\htdocs\yii\framework\db\schema\CDbCommandBuilder.php:828
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\schema\CDbCommandBuilder.php(621): CDbCommandBuilder->createInCondition(Object(CMysqlTableSchema), NULL, Array, '`t`.')
#1 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1485): CDbCommandBuilder->createPkCriteria(Object(CMysqlTableSchema), '1', '', Array, '`t`.')
#2 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(48): CActiveRecord->findByPk('1')
#3 [internal function]: DienteController->actionUpdate('1')
#4 M:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(DienteController), Array)
#5 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(DienteController), Object(ReflectionMethod), Array)
#6 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#10 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/update')
#11 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/yii/facilito/diente/update/1.html
HTTP_REFERER=http://localhost/yii/facilito/diente/
---
2014/04/28 21:35:51 [error] [exception.CDbException] exception 'CDbException' with message 'Column name must be either a string or an array.' in M:\xampp\htdocs\yii\framework\db\schema\CDbCommandBuilder.php:828
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\schema\CDbCommandBuilder.php(621): CDbCommandBuilder->createInCondition(Object(CMysqlTableSchema), NULL, Array, '`t`.')
#1 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1485): CDbCommandBuilder->createPkCriteria(Object(CMysqlTableSchema), '1', '', Array, '`t`.')
#2 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(48): CActiveRecord->findByPk('1')
#3 [internal function]: DienteController->actionUpdate('1')
#4 M:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(DienteController), Array)
#5 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(DienteController), Object(ReflectionMethod), Array)
#6 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#10 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/update')
#11 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/yii/facilito/diente/update/1.html
HTTP_REFERER=http://localhost/yii/facilito/diente/
---
2014/04/28 21:36:02 [error] [exception.CDbException] exception 'CDbException' with message 'Column name must be either a string or an array.' in M:\xampp\htdocs\yii\framework\db\schema\CDbCommandBuilder.php:828
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\schema\CDbCommandBuilder.php(621): CDbCommandBuilder->createInCondition(Object(CMysqlTableSchema), NULL, Array, '`t`.')
#1 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1485): CDbCommandBuilder->createPkCriteria(Object(CMysqlTableSchema), '1', '', Array, '`t`.')
#2 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(48): CActiveRecord->findByPk('1')
#3 [internal function]: DienteController->actionUpdate('1')
#4 M:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(DienteController), Array)
#5 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(DienteController), Object(ReflectionMethod), Array)
#6 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#10 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/update')
#11 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/yii/facilito/diente/update/1.html
HTTP_REFERER=http://localhost/yii/facilito/diente/
---
2014/04/28 21:36:05 [error] [exception.CDbException] exception 'CDbException' with message 'Column name must be either a string or an array.' in M:\xampp\htdocs\yii\framework\db\schema\CDbCommandBuilder.php:828
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\schema\CDbCommandBuilder.php(621): CDbCommandBuilder->createInCondition(Object(CMysqlTableSchema), NULL, Array, '`t`.')
#1 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1485): CDbCommandBuilder->createPkCriteria(Object(CMysqlTableSchema), '0', '', Array, '`t`.')
#2 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(48): CActiveRecord->findByPk('0')
#3 [internal function]: DienteController->actionUpdate('0')
#4 M:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(DienteController), Array)
#5 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(DienteController), Object(ReflectionMethod), Array)
#6 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#10 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/update')
#11 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/yii/facilito/diente/update/0.html
HTTP_REFERER=http://localhost/yii/facilito/diente/
---
2014/04/28 21:36:07 [error] [exception.CDbException] exception 'CDbException' with message 'Column name must be either a string or an array.' in M:\xampp\htdocs\yii\framework\db\schema\CDbCommandBuilder.php:828
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\schema\CDbCommandBuilder.php(621): CDbCommandBuilder->createInCondition(Object(CMysqlTableSchema), NULL, Array, '`t`.')
#1 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1485): CDbCommandBuilder->createPkCriteria(Object(CMysqlTableSchema), '0', '', Array, '`t`.')
#2 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(48): CActiveRecord->findByPk('0')
#3 [internal function]: DienteController->actionUpdate('0')
#4 M:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(DienteController), Array)
#5 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(DienteController), Object(ReflectionMethod), Array)
#6 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#10 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/update')
#11 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/yii/facilito/diente/update/0.html
HTTP_REFERER=http://localhost/yii/facilito/diente/
---
2014/04/28 21:36:09 [error] [exception.CDbException] exception 'CDbException' with message 'Column name must be either a string or an array.' in M:\xampp\htdocs\yii\framework\db\schema\CDbCommandBuilder.php:828
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\schema\CDbCommandBuilder.php(621): CDbCommandBuilder->createInCondition(Object(CMysqlTableSchema), NULL, Array, '`t`.')
#1 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1485): CDbCommandBuilder->createPkCriteria(Object(CMysqlTableSchema), '0', '', Array, '`t`.')
#2 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(48): CActiveRecord->findByPk('0')
#3 [internal function]: DienteController->actionUpdate('0')
#4 M:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(DienteController), Array)
#5 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(DienteController), Object(ReflectionMethod), Array)
#6 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#10 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/update')
#11 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/yii/facilito/diente/update/0.html
HTTP_REFERER=http://localhost/yii/facilito/diente/
---
2014/04/28 21:36:11 [error] [exception.CDbException] exception 'CDbException' with message 'Column name must be either a string or an array.' in M:\xampp\htdocs\yii\framework\db\schema\CDbCommandBuilder.php:828
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\schema\CDbCommandBuilder.php(621): CDbCommandBuilder->createInCondition(Object(CMysqlTableSchema), NULL, Array, '`t`.')
#1 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1485): CDbCommandBuilder->createPkCriteria(Object(CMysqlTableSchema), '1', '', Array, '`t`.')
#2 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(48): CActiveRecord->findByPk('1')
#3 [internal function]: DienteController->actionUpdate('1')
#4 M:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(DienteController), Array)
#5 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(DienteController), Object(ReflectionMethod), Array)
#6 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#10 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/update')
#11 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/yii/facilito/diente/update/1.html
HTTP_REFERER=http://localhost/yii/facilito/diente/
---
2014/04/28 21:36:13 [error] [exception.CDbException] exception 'CDbException' with message 'Column name must be either a string or an array.' in M:\xampp\htdocs\yii\framework\db\schema\CDbCommandBuilder.php:828
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\schema\CDbCommandBuilder.php(621): CDbCommandBuilder->createInCondition(Object(CMysqlTableSchema), NULL, Array, '`t`.')
#1 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1485): CDbCommandBuilder->createPkCriteria(Object(CMysqlTableSchema), '2', '', Array, '`t`.')
#2 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(48): CActiveRecord->findByPk('2')
#3 [internal function]: DienteController->actionUpdate('2')
#4 M:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(DienteController), Array)
#5 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(DienteController), Object(ReflectionMethod), Array)
#6 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#10 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/update')
#11 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/yii/facilito/diente/update/2.html
HTTP_REFERER=http://localhost/yii/facilito/diente/
---
2014/04/28 21:36:16 [error] [exception.CDbException] exception 'CDbException' with message 'Column name must be either a string or an array.' in M:\xampp\htdocs\yii\framework\db\schema\CDbCommandBuilder.php:828
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\schema\CDbCommandBuilder.php(621): CDbCommandBuilder->createInCondition(Object(CMysqlTableSchema), NULL, Array, '`t`.')
#1 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1485): CDbCommandBuilder->createPkCriteria(Object(CMysqlTableSchema), '2', '', Array, '`t`.')
#2 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(48): CActiveRecord->findByPk('2')
#3 [internal function]: DienteController->actionUpdate('2')
#4 M:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(DienteController), Array)
#5 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(DienteController), Object(ReflectionMethod), Array)
#6 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#7 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#8 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#10 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/update')
#11 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/yii/facilito/diente/update/2.html
HTTP_REFERER=http://localhost/yii/facilito/diente/
---
2014/04/28 21:40:05 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in M:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 M:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#4 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/update')
#5 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/yii/facilito/diente/update/
---
2014/04/28 21:40:07 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Your request is invalid.' in M:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 M:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#4 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/update')
#5 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/yii/facilito/diente/update
---
2014/04/28 21:41:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "delete".' in M:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 M:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('delete')
#1 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('delete')
#2 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/delete')
#3 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii/facilito/diente/delete/1.html
HTTP_REFERER=http://localhost/yii/facilito/diente/
---
2014/04/28 21:45:20 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '0' for key 'PRIMARY'. The SQL statement executed was: INSERT INTO `diente` (`nombre`, `estado`) VALUES (:yp0, :yp1).
2014/04/28 21:45:20 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '0' for key 'PRIMARY'' in M:\xampp\htdocs\yii\framework\db\CDbCommand.php:358
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1077): CDbCommand->execute()
#1 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(806): CActiveRecord->insert(NULL)
#2 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(33): CActiveRecord->save()
#3 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionCreate()
#4 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#8 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/create')
#9 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii/facilito/diente/create.html
HTTP_REFERER=http://localhost/yii/facilito/diente/create.html
---
2014/04/28 21:45:26 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '0' for key 'PRIMARY'. The SQL statement executed was: INSERT INTO `diente` (`nombre`, `estado`) VALUES (:yp0, :yp1).
2014/04/28 21:45:26 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '0' for key 'PRIMARY'' in M:\xampp\htdocs\yii\framework\db\CDbCommand.php:358
Stack trace:
#0 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1077): CDbCommand->execute()
#1 M:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(806): CActiveRecord->insert(NULL)
#2 M:\xampp\htdocs\yii\facilito\protected\controllers\DienteController.php(33): CActiveRecord->save()
#3 M:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): DienteController->actionCreate()
#4 M:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 M:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 M:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#8 M:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('diente/create')
#9 M:\xampp\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 M:\xampp\htdocs\yii\facilito\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii/facilito/diente/create.html
HTTP_REFERER=http://localhost/yii/facilito/diente/create.html
---
